lib/src/main/java/com/diffplug/spotless/java/ImportOrderStep.java

```diff
@@ -39,6 +40,9 @@
 
 public final class ImportOrderStep {
 	private static final boolean WILDCARDS_LAST_DEFAULT = false;
+	private static final boolean SEMANTIC_SORT_DEFAULT = true;
```

Comment:
We should not change the default behavior for existing users of the `importOrder` step.